zlibWrapper/examples/zwrapbench.c

```diff
@@ -130,7 +130,7 @@ void BMK_SetBlockSize(size_t blockSize)
 **********************************************************/
 typedef struct
 {
-    const char* srcPtr;
+    char* srcPtr;
```

Comment:
A minor question : 
it's not a big deal since `zwrapbench` is a test program,
but is there any specific reason you need `srcPtr` to be non-`const` ?